index.d.ts

```diff
@@ -288,6 +288,21 @@ interface DocumentEnd {
   append(content: Content, options?: ContentOptions): DocumentEnd;
 }
 
+type DurableObjectLocationHint =
+  | "afr"
```

Comment:
Does this work with the initial `|`? I've never seen this before.

Reply:
FWIW, yeah. This is result of (mandatory) prettier. :dizzy_face: